app/src/main/java/ch/epfl/sdp/item/Coin.java

```diff
@@ -46,7 +46,7 @@ public double getValue() {
     public void update() {
         if (!taken) {
             Player p = PlayerManager.getInstance().getCurrentUser();
-            if (this.location.distanceTo(p.getLocation()) - p.getAoeRadius() >= 1) {
+            if (this.location.distanceTo(p.getLocation()) - p.getAoeRadius() - 50 >= 1) {
```

Comment:
it's a good idea to remove magic constants like 50

Reply:
I changed it

Reply:
Thanks for the comment